### ISSUE TITLE
Automatically release to the vscode extension store

### ISSUE THREAD

**Issue by Southclaws** [Southclaws/vscode-pawn#44]
@ADRFranklin do you know if there's a GitHub action we can easily set up for this and trigger it on tagged pushes to master?

**Comment by ADRFranklin** [comment]
https://github.com/marketplace/actions/publish-vs-code-extension

this?

**Comment by Southclaws** [comment]
Yeah that looks good, we can pair on this some time, I'll have to add my vscode extension store access token to the github actions secrets too.

**Comment by ADRFranklin** [comment]
@Southclaws alright, looking forward to the pair sesh.
